src/examples/java/com/amazonaws/crypto/examples/DataKeyCaching/cloudformation-template.yaml

```diff
@@ -0,0 +1,221 @@
+Parameters:
```

Comment:
burying this file under a deep path where people are probably only expecting java source is probably less than ideal